src/components/Login/Login.js

```diff
@@ -12,11 +12,10 @@ class Login extends Component {
         this.props.history.push('/dashboard');
       }
     } catch (e) {
-      console.log(e);
+      console.log('Caught error in Login');
```

Comment:
you might want to avoid console.log statements in master

Reply:
Done!